.gitattributes

```diff
@@ -1,4 +1,7 @@
+.phpstan/ export-ignore
 .* export-ignore
 *.md export-ignore
 tests export-ignore
 docs export-ignore
+Makefile export-ignore
```

Comment:
I forgot , this file is edited by dev kit, it cannot be done manually here, please revert changes on this file.

It should be edited here: https://github.com/sonata-project/dev-kit/blob/master/project/.gitattributes.twig